kubel.el

```diff
@@ -839,6 +857,7 @@ RESET is to be called if the search is nil after the first attempt."
    ("l" "Logs" kubel-log-popup)
    ("c" "Copy" kubel-copy-popup)
    ("e" "Exec" kubel-exec-pod)
+   ("b" "Shell" kubel-exec-shell-pod)
```

Comment:
I'm not a fan of the `b` here. I think it would make more sense to create a `kubel-exec-popup` similar to `kubel-log-popup`. The popup would be bound to `e` and would have two options:
1. `kubel-exec-pod` => `e`
2. `kubel-exec-shell-pod` => `b` (if that's still what you want, but maybe `s` for shell also makes sense idk)

It's technically a breaking change and I would have to cut a new a release but I think it's more intuitive and follows the general magit binding conventions.

Reply:
Also, if you do not want to bother, just don't bind your `kubel-exec-shell-pod` for now and I will make the change after merge. Totally up to you

Reply:
I added `kubel-exec-popup` menu. Could you have a look?